packages/flutter_tools/test/general.shard/build_system/targets/macos_test.dart

```diff
@@ -131,7 +131,7 @@ void main() {
       ..createSync(recursive: true)
       ..writeAsStringSync('testing');
 
-    await const DebugBundleFlutterAssets().build(<File>[], environment);
+    await const DebugMacOSBundleFlutterAssets().build(<File>[], environment);
```

Comment:
Shouldn't there be new tests (in both files) exercising profile and release?

Reply:
I've added a test that exercises release, though it is mostly mocks.